### ISSUE TITLE
Creation of Multi-sig wallet hangs on "creating"

### ISSUE THREAD

**Issue by dreamer4031** [ethereum/mist#3844]
* [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

![image](https://user-images.githubusercontent.com/38547872/39028503-f8f82af6-4491-11e8-9fa4-5c9b50420799.png)
I have been trying to create a multi-sig wallet. The first one was a trial one. The second and the third one are still "creating" and hangs there. The second one I didn't have enough ETH to fund the creation, so I put in some ETH and created the third one "RootProject Multisig (final)" but both of them hang on "creating." 

about 0.5 ETH has gone out from my account. 
Resolution please.

Thanks,

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0`
Number of blocks synchronized: 0
```

**Comment by ChristBKK** [comment]
Do you run the Light Client? Doesn't work for me for months in the Light Client. I run the full synced Mist and it works fine.  Takes days to sync but then it's fine :)

**Comment by ryanio** [comment]
Hey, we just released a new version (`0.11.0`) - please give it a try. You may need to copy down the details of your multisigs first to bring them into the new version. If you have any new problems that you can reproduce, please create a new issue with the details on how we can help. Thanks!
